include/clap/ext/draft/scratch-memory.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include "../../plugin.h"
+
+// This extension lets the plugin request "scratch" memory
+// from the host. Scratch memory can be accessed during the
+// `process()` callback, but is not persistent between callbacks.
```

Comment:
"but it's content isn't persistent between callbacks."

It could be confused with it's allocation that isn't persistent.